tests/MySqlConnector.Tests/MySqlDecimalTests.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using Xunit;
+
+namespace MySqlConnector.Tests;
+public class MySqlDecimalTests
+{
+#if !BASELINE
```

Comment:
I missed earlier that MySql.Data has no public constructor for this type, so that there's no way to test it. In that case, the entire file can be excluded in the csproj instead of `#if`'ed out here.